src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -372,6 +374,33 @@ public static function underscore($id)
         return strtolower(preg_replace(array('/([A-Z]+)([A-Z][a-z])/', '/([a-z\d])([A-Z])/'), array('\\1_\\2', '\\1_\\2'), str_replace('_', '.', $id)));
     }
 
+    /**
+     * Fetches a variable from the environment.
+     *
+     * @param string The name of the environment variable
+     *
+     * @return scalar The value to use for the provided environment variable name
+     *
+     * @throws EnvNotFoundException When the environment variable is not found and has no default value
+     */
+    protected function getEnv($name)
```

Comment:
Yeah actually I want to get rid of `app_dev.php` as well so using dotenv, although doable is a bit annoying (dotenv should not be used in production).

But the answers that have been given are quite helpful, thanks for it :)